circus-train-api/src/main/java/com/hotels/bdp/circustrain/api/metastore/MetaStoreClientException.java

```diff
@@ -1,27 +0,0 @@
-/**
- * Copyright (C) 2016-2017 Expedia Inc.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- * http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-package com.hotels.bdp.circustrain.api.metastore;
-
-import com.hotels.bdp.circustrain.api.CircusTrainException;
-
-public class MetaStoreClientException extends CircusTrainException {
```

Comment:
Did you check all places where we catch `CircusTrainException` and made sure we don't miss this exception anymore?

Reply:
It looks like we are ok, we only catch `CircusTrainException` in 6 places, 5 of which are tests

Reply:
No